samplers/jaegerremote/sampler_remote.go

```diff
@@ -151,20 +150,20 @@ func (s *Sampler) setSampler(sampler trace.Sampler) {
 func (s *Sampler) UpdateSampler() {
 	res, err := s.samplingFetcher.Fetch(s.serviceName)
 	if err != nil {
-		// log.Printf("failed to fetch sampling strategy: %v", err)
+		s.logger.Error(err, "failed to fetch sampling strategy")
 		return
 	}
 	strategy, err := s.samplingParser.Parse(res)
 	if err != nil {
-		// log.Printf("failed to parse sampling strategy response: %v", err)
+		s.logger.Error(err, "failed to parse sampling strategy response")
 		return
 	}
 
 	s.Lock()
 	defer s.Unlock()
 
 	if err := s.updateSamplerViaUpdaters(strategy); err != nil {
-		//c.logger.Infof("failed to handle sampling strategy response %+v. Got error: %v", res, err)
+		s.logger.Error(err, fmt.Sprintf("failed to handle sampling strategy response %+v.", res))
```

Comment:
Use the structure nature of the logger. This allows logging implementation to determine how they want to display this information.

```suggestion
		s.logger.Error(err, "failed to handle sampling strategy response", "response", res)
```